openapi/workflow_execution_service.swagger.yaml

```diff
@@ -264,16 +274,26 @@ definitions:
         title: The command line that was run
       start_time:
         type: string
-        title: When the command was executed
+        title: When the command started executing, in ISO 8601 format "%Y-%m-%dT%H:%M:%SZ"
       end_time:
         type: string
-        title: When the command completed
+        title: When the command stopped executing (completed, failed, or cancelled), in ISO 8601 format "%Y-%m-%dT%H:%M:%SZ"
       stdout:
         type: string
-        title: Sample of stdout (not guaranteed to be entire log)
+        title: |-
+          A URL to retrieve standard output logs of the workflow run or
```

Comment:
Might be useful to provide client some indication of what protocol is required to access log URLs (e.g., http, gs, s3, etc.). I'm thinking that `supported_filesystem_protocols` in `ServiceInfo` should be a map instead of an array (e.g., `{"workflows": ["http", "file"], "tasks": ["gs"], "logs": ["gs"]}`) — but I'll open a separate issue to discuss.